packages/core/src/vector/libsql/index.ts

```diff
@@ -239,6 +248,7 @@ export class LibSQLVector extends MastraVector {
 
   async deleteIndex(indexName: string): Promise<void> {
     try {
+      validateSqlIdentifier(indexName, 'index name');
       // Drop the table
       await this.turso.execute({
         sql: `DROP TABLE IF EXISTS ${indexName}`,
```

Comment:
Not a huge deal to actually do this in this PR but this is a good pattern https://lexi-lambda.github.io/blog/2019/11/05/parse-don-t-validate/

In practice it would look like this:
```suggestion
      // Drop the table
      await this.turso.execute({
        sql: `DROP TABLE IF EXISTS ${parseSqlIdentifier(indexName, 'index name')}`,
```

Then you get type safety because TS can see that if an input doesn't match some condition it will throw, and it's able to narrow types. In this case the type wouldn't be narrowed more so it doesn't matter too much - but it's generally a good pattern to get in the habit of using

Reply:
```ts
const parseSqlIdentifier = (input: X) => X is Y {
  if (typeof input !== `something`) throw new Error()
  return input
}
```
then TS knows more about the return value

Reply:
added parse functions instead of validate